### DISCUSSION TITLE
Sugerencia para bajar el multiplicador de Flower Power

### DISCUSSION THREAD

**Discussion by Macacosky** [Re-Volt-America/rva_pack#9] [RVA Cars]
Solicitud de reducción al multiplicador de Flower Power, actualmente 1.2, dados los siguientes puntos:
- El auto es sencillamente más viable a la hora de conducir que Groovster, auto con menor multiplicador (1.1) y peor desempeño en carreras de pick ups.
- Puede lograr posiciones altas si se sabe conducir, dado a que tiene una buena aceleración con respecto a la clase, un buen radio de giro y conservación de la velocidad mientras está doblando.
- Si bien el auto tiene bastantes inconsistencias, es perfectamente manejable para los mapas que hoy en día se juegan en RVA, en la mayoría de estos no tiene problemas, (lo he jugado dentro de sesiones, y si bien, hay que ser cuidadoso en ciertas partes, se pueden lograr buenas cosas con el auto) pudiendo rivalizar en bastantes oportunidades con autos de multiplicador inferior.
- Mi sugerencia de multiplicador sería 1.1 o 1.0

**Comment by BGMP** [comment]
This sugestion has been accepted! For reference, here are the changes for the Advanced class multipliers:

**Advanced**
Groovster 1.1 ➔ 1.2
Shocker 1.0 ➔ 1.1
Emperor 1.0 ➔ 1.1

Artctic Stomper 1.0 ➔ 0.8
Flower Power 1.2 ➔ 1.0
Bertha Ballistics 0.9 ➔ 0.8
